src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -749,24 +749,26 @@ export class ChatWidget extends Disposable implements IChatWidget {
 			// TODO@bhavyaus remove this startup experiment once settled
 			const startupExpValue = startupExpContext.getValue(this.contextKeyService);
 			const configuration = this.configurationService.inspect('workbench.secondarySideBar.defaultVisibility');
-			const expIsActive = configuration.defaultValue !== 'hidden';
+			const expIsActive = configuration.value === 'hidden';
```

Comment:
@justschen we're showing the new experimental welcome view as a part of @bpasero 's experiment to show chat panel on workspace open for the first time. This change would override that? Is this intentional?